sample-xml/build.gradle.kts

```diff
@@ -32,7 +32,15 @@ tasks.register("clean", Delete::class) {
 
 tasks.withType<io.gitlab.arturbosch.detekt.Detekt>().configureEach {
     // Target version of the generated JVM bytecode. It is used for type resolution.
-    jvmTarget = JavaVersion.VERSION_1_8.toString()
+    jvmTarget = JavaVersion.VERSION_11.toString()
+    reports {
+        xml {
+            outputLocation.set(file("build/reports/detekt/detekt.xml"))
+        }
+        html {
+            outputLocation.set(file("build/reports/detekt/detekt.html"))
+        }
+    }
```

Comment:
I think you forgot to remove the duplicated `reports` config at [lines 66-75](https://github.com/nimblehq/android-templates/pull/347/files#diff-a8284819013df74f466c134a33f1a68a01b63ed7772101bca2fb655606ab9a17L58-L67).